### PR TITLE
FEAT: add link to OSSPH logo

### PR DESCRIPTION
**Pull request by AbbeyIT**
## Fixes
Issue #53 

## Details
On the homepage of [Freefolio](https://ossphilippines.github.io/freefolio/), I added the [OSSPH](https://ossph.org/) link in the logo.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
     <header class="bg-violet-500">
       <div class="container mx-auto px-5 py-8 flex flex-row justify-between">
-        <img class="h-14" src="./images/osslogo-text-white.png" alt="">
+        <a href="https://ossph.org/" target="_blank"><img class="h-14" src="./images/osslogo-text-white.png" alt=""></a>
```
Comment (jofftiquez) [inline index.html:18]: Hi, all good. I just have one last request. Please add "OSSPH Logo" as `alt` in the image. Thanks.

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (AbbeyIT) [reply index.html]: Done. Kindly please review it, Thank you.

**Review by jofftiquez — APPROVED** [review]
Perfect! Thanks!

**Comment by AbbeyIT** [comment]
Thank you @jofftiquez

**Comment by mpotane** [comment]
Closes #53

**Pull request merged**